src/templates/actionBar.jsx

```diff
@@ -136,185 +164,217 @@ class ActionBar extends React.PureComponent {
-                                                                className="action_bar--search_icon"
-                                                                color={isMobileSearchVisible ?
-                                                                    'highlight' :
-                                                                    null}
-                                                                id={iconSearch.id}
-                                                                onClick={this.onMobileSearchIconToggle}
-                                                                title={iconSearch.title || 'Search'}
-                                                                type="search"
-                                                            />
-                                                            )}
-                                                            {/* eslint-enable max-len */}
-
-                                                            {iconTable && (
-                                                            <Icon
-                                                                color={iconTable.selected ?
-                                                                    'highlight' :
-                                                                    null}
-                                                                id={iconTable.id}
-                                                                onClick={iconTable.onClick}
-                                                                title={iconTable.title || 'Table View'}
-                                                                type="list"
-                                                            />
-                                                            )}
+                                                    })}
+                                                </DropdownButton>
+                                            )}
 
-                                                            {jsx}
-                                                        </List.Item>
-                                                    );
-                                                })}
-                                            </List>
+                                            {list && _.isArray(list) && (
+                                                <List
+                                                    className="action_bar--list"
+                                                    horizontal
+                                                >
+                                                    {_.map(list, (item) => {
+                                                        const {
+                                                            actionsButton,
+                                                            jsx, // eslint-disable-line no-shadow
```

Comment:
Hmm, are we sure this is okay?  This is one where (despite my rebelliousness mentioned above) I'd probably heed the warning and maybe alias this as `innerJsx` or some such to avoid the variable shadowing.  That's one of those things in my book where the warning is correct in telling you you might be doing something dangerous; think twice.  If it's correct, and you want to leave it as-is, no objections.  NAB.

Reply:
Oh, let me get this one.